lib/modules/datasource/maven/util.ts

```diff
@@ -485,6 +485,18 @@ export async function getDependencyInfo(
     }
   }
 
+  const relocation = pomContent.descendantWithPath(
+    'distributionManagement.relocation',
+  );
```

Comment:
The distributionManagement.relocation settings can also be configured inside a profile. Did you consider supporting that too?